products/compute/api.yaml

```diff
@@ -935,6 +935,266 @@ objects:
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: 'includeHost'
+                description: |
+                  If true requests to different hosts will be cached separately.
+              - !ruby/object:Api::Type::Boolean
+                name: 'includeProtocol'
+                description: |
+                  If true, http and https requests will be cached separately.
+              - !ruby/object:Api::Type::Boolean
+                name: 'includeQueryString'
+                description: |
+                  If true, include query string parameters in the cache key
+                  according to query_string_whitelist and
+                  query_string_blacklist. If neither is set, the entire query
+                  string will be included.
+
+                  If false, the query string will be excluded from the cache
+                  key entirely.
+              - !ruby/object:Api::Type::Array
+                name: 'queryStringBlacklist'
+                description: |
+                  Names of query string parameters to exclude in cache keys.
+
+                  All other parameters will be included. Either specify
+                  query_string_whitelist or query_string_blacklist, not both.
+                  '&' and '=' will be percent encoded and not treated as
+                  delimiters.
+                item_type: Api::Type::String
+              - !ruby/object:Api::Type::Array
+                name: 'queryStringWhitelist'
+                description: |
+                  Names of query string parameters to include in cache keys.
+
+                  All other parameters will be excluded. Either specify
+                  query_string_whitelist or query_string_blacklist, not both.
+                  '&' and '=' will be percent encoded and not treated as
+                  delimiters.
+                item_type: Api::Type::String
+          - !ruby/object:Api::Type::Integer
+            name: 'signedUrlCacheMaxAgeSec'
+            default_value: 3600
+            description: |
+              Maximum number of seconds the response to a signed URL request
+              will be considered fresh, defaults to 1hr (3600s). After this
+              time period, the response will be revalidated before
+              being served.
+
+              When serving responses to signed URL requests, Cloud CDN will
+              internally behave as though all responses from this backend had a
+              "Cache-Control: public, max-age=[TTL]" header, regardless of any
+              existing Cache-Control header. The actual headers served in
+              responses will not be altered.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'connectionDraining'
+        description: |
+          Settings for connection draining
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'drainingTimeoutSec'
+            default_value: 0
```

Comment:
This defaults to 0 to keep with current functionality

Reply:
We can do this as an override in TF so Ansible / Inspec infer correct values later on if you'd like as well.